src/schema/config.schema.ts

```diff
@@ -0,0 +1,43 @@
+import * as Joi from "@hapi/joi";
+
+let environment = "development";
+if (process.env.NODE_ENV) {
+  environment = process.env.NODE_ENV;
+}
+
+let configValidation;
+
+if (["staging", "production"].includes(environment)) {
+  configValidation = {
+    DATABASE_URL: Joi.string().required(),
+    SENTRY_DNS: Joi.string().required(),
+    JWT_SECRET_KEY: Joi.string().required(),
+    JWT_EXPIRES_TIME: Joi.string().required(),
+    AWS_REGION: Joi.string().required(),
+    AWS_ACCESS_KEY_ID: Joi.string().required(),
+    AWS_SECRET_ACCESS_KEY: Joi.string().required(),
+    AWS_PUBLIC_BUCKET_NAME: Joi.string().required(),
+    EMAIL_VERIFY_OTP: Joi.string().required(),
+    EMAIL_PASSWORD_VERIFY_OTP: Joi.string().required(),
```

Comment:
they have some common config, like EMAIL_VERIFY_OTP and EMAIL_PASSWORD_VERIFY_OTP so let create a common config first